change/@ni-nimble-components-9da2496e-e544-4019-a666-d7fca3046771.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm seeing a bug when I play with this in storybook:
1. Open storybook for the select with a placeholder & filter-mode set to `standard`
2. Using the mouse, click the select to open it and then click "Option 3"
3. Using the mouse, click the select to open it again
4. Type "Zur" in the filter box
5. Press Escape to close the dropdown
6. Press Enter to open the dropdown again
7. Notice that the value in the select incorrectly updates to "Zurich" and "Option 3" is selected with the green highlight
8. Also notice that if you press the Up Arrow, "Option 4" becomes highlighted in green in addition to "Option 3" staying highlighted.

Reply:
This issue stemmed from the typeahead feature of the `Select`. My solution was to fork/override the relevant implementation from the FAST `Listbox` so that we can control when the `selectedIndex` is being updated, and additionally manage the selected state of the options as we do elsewhere while the dropdown is open.